docs/faq.md

```diff
@@ -161,13 +160,43 @@ See [the Semgrep development philosophy](/contributing/semgrep-philosophy/) for
 
 ### Comparing Semgrep to Snyk
 
-For SCA, reachability analysis is critical to cut down noise and reduce false positives. Semgrep offers reachability analysis for [several languages, such as Java, JavaScript, and Ruby](supported-languages/#semgrep-supply-chain) whereas Snyk only offers reachability for Java.
+#### SAST
+
+Both Semgrep and Snyk offer SAST solutions that are out of the box. Semgrep makes it easier to customize the rules that run against your code. Furthermore, when you know which rules were run, you can analyze your results to see if the relevant vulnerabilities were caught.
```

Comment:
```suggestion
Both Semgrep and Snyk offer out-of-the-box SAST and SCA solutions. Semgrep makes it easier to customize the rules that run against your code. Because these rules are visible and customizable, you can analyze your results to see if the relevant vulnerabilities were caught.
```